src/componmnet/PointCloud/Panel.jsx

```diff
@@ -3,16 +3,27 @@ import './Panel.css';
 import { DownOutlined } from '@ant-design/icons';
 import { Menu, Dropdown, Button, AutoComplete, Slider, Input} from 'antd';
 
-function Panel({scene}) {
+function Panel({display, setDisplay, setDisplay2D, buttonLabel, setLabel, width}) {
 
     return (
         <div className='panel panel-3d'>
             <div className='panel-head-3d'>
-                <h1>Netowrk</h1>
+                <h1>Network</h1>
 
-                <a>Switch to {scene === 0  ? '2D' : '3D' }</a>
+                <a onClick={() => {
+                    if (display === "") {
+                        setDisplay("none");
+                        setDisplay2D("");
+                        setLabel("3D");
+                    } else {
+                        setDisplay("");
+                        setDisplay2D("none");
+                        setLabel("2D");
+                    }
+                }}> 
+                Switch to {buttonLabel}</a>
             </div>
-            <div className='panel-body-3d'>
+            <div className='panel-body-3d' style={{width: width}}>
```

Comment:
Should be in css file

Reply:
this is a variable from javascript, not sure how to pass this into css file